docs/integrations/editors.md

```diff
@@ -160,6 +160,16 @@ If you need to do anything special to make your virtualenv work and install _Bla
 example you want to run a version from main), create a virtualenv manually and point
 `g:black_virtualenv` to it. The plugin will use it.
 
+If you would prefer to use the system installation of _Black_ rather than a virtualenv,
+then add this to your vimrc:
+
+```
+let g:black_use_system = 1
```

Comment:
```suggestion
let g:black_system_install = 1
```
Or something, "use system" isn't immediately that obvious

Reply:
Yes, naming is hard. Would it be better to have this be `g:black_use_virtualenv` and set the default to `1` (so the actual default behavior remains unchanged)?

Reply:
Sounds good 👍

Reply:
Ok, I updated the PR and re-tested it locally. This still seems to work for me, with the default behavior being the same.

I also added fleshed out the error message and the documentation to (hopefully) be a bit more helpful.

Reply:
...and rebased now to fix merge conflict.